elastic-tls/generate-tls.sh

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+declare ELASTIC_VERSION="${ELASTIC_VERSION:-7.17.0}"
+declare TYPE="${TYPE:-cluster}"
+
+check_docker() {
+  if ! command -v docker &> /dev/null
+  then
+    echo "Docker could not be found"
+    echo "Docker is required to run this script, please install"
+    exit
+  fi
+}
+
+check_config() {
+  if [ ! -f ./certutil-input.yaml ]; then
+    echo "certutil-input.yaml not found"
+    echo "You must create certutil-input.yaml before running this script."
+    exit
+  fi
+}
+
+help() {
+   # Display Help
+   echo "Script for generating Elastic TLS files."
+   echo
+   echo "Usage: $0 [-v <string>] [-t <single|cluster>] [-p <string>]"
+   echo
+   echo "options:"
+   echo 
+   echo "-v     Set Elaticsearch version, exp 7.17.0."
+   echo "-t     single or cluster type."
+   echo "-h     Print Help."
+   echo
+}
+
+usage() { help 1>&2; exit 1; }
+
+clean() {
+
+  if [ -f ./my-ca.p12 ]; then
+    echo "Clean up old CA / Certificates"
+    rm -rf ./my-ca.p12 ./my-keystore.p12 ./keystore ./hsperfdata_root
+  fi
+  
+}
+
+generate_ca() {
+  docker run --rm -t \
+    --mount type=bind,source="$(pwd)",target=/tmp \
+    docker.elastic.co/elasticsearch/elasticsearch:"${ELASTIC_VERSION}" \
+    bash -c " elasticsearch-certutil ca --silent --out /tmp/my-ca.p12 --pass \"\""
```

Comment:
I would suggest adding one:

```suggestion
    bash -c " elasticsearch-certutil ca --silent --out /tmp/my-ca.p12 --pass \"${ES_CA_CERT_PASSWORD}\""
```